include/cadmium/logger/common_loggers_helpers.hpp

```diff
@@ -0,0 +1,97 @@
+#ifndef CADMIUM_COMMON_LOGGERS_HELPERS_HPP
```

Comment:
Please add license here

Reply:
Done.

Reply:
Should I put both names or keep  only yours? 
This question applies to all the files where there is code from both of us, should I add my name there?